e2e/src/main/java/gov/hhs/cdc/trustedintermediary/e2e/SentPayloadReader.java

```diff
@@ -14,13 +14,7 @@ public static String read() throws IOException {
 
     private static Path findFilePayload() {
```

Comment:
If we're not using this method anywhere else, we probably don't need the `SentPayloadReader` class. We could just move the file opening line to the test where it's called. That way we don't have unnecessary complexity and it would make the test more readable